### ISSUE TITLE
Add version number to "latest"

### ISSUE THREAD

**Issue by akeller** [camunda/camunda-platform-docs#587]
```
versions: {
            current: {
              label: `latest (1.3)`,
            },
          },
```

This change would update the badges and top-level navigation. This is also in the docusaurus.config.js file, somewhat near the bottom, so rather than hardcoding a version value when cutting a new release, it would be great to include a variable or something that autoincrements for less user error.

**Comment by akeller** [comment]
@pepopowitz FYI since you'll be working on versioning changes. My guess is this will be moot with those changes, and if so please close it.

**Comment by pepopowitz** [comment]
@akeller I'm looking for clarification/confirmation that the changes demonstrated in #910 will give you the result you're looking for. I _think_ that they will, but want to point out a couple details to be sure -- 

1. There will be no "Latest" option in the version dropdown. There will instead be the latest version number, e.g. `1.3`. 
2. There will be a new "Next" option in the version dropdown. Though I think we can call this version whatever we want in the docusuarus configuration, if we don't like that name. 

Does this introduction of a "Next" option without a version number attached to it re-introduce the same problem/concerns as prompted this issue?

**Comment by akeller** [comment]
#910 is exactly what we are looking to address here. If you look at C7 docs, you'll see our current users are familiar with a mix of numbered and named versions (develop, latest).

**Comment by pepopowitz** [comment]
Ah, cool. Thank you! I agree then that this issue is unneeded, since it will be resolved with the changes demonstrated in #910. Closing!
